src/components/Garden/DecisionLoader.tsx

```diff
@@ -1,9 +1,15 @@
 import React from 'react'
 import DecisionDetail from './DecisionDetail/DecisionDetail'
 import { GardenLoader } from '../Loader'
-import useProposalLogic from '../../logic/proposal-logic'
+import useProposalLogic, {
+  ProposalLogicProps,
+} from '../../logic/proposal-logic'
```

Comment:
```suggestion
import useProposalLogic, { ProposalLogicProps } from '@/logic/proposal-logic'
```

Suggestion of use @ instead ../../ to point to src folder.